src/lib/components/AnnouncementBanner.svelte

```diff
@@ -1,8 +1,19 @@
 <script lang="ts">
     import { page } from '$app/stores';
+
+    function isDateSameOrAfter(date: Date) {
+        const currentDate = new Date();
+        currentDate.setHours(0, 0, 0, 0);
+        date.setHours(0, 0, 0, 0);
+
+        return date >= currentDate;
```

Comment:
Hopefully this won't be problematic due to time zones 😬

Reply:
Good workaround?